web-client/src/models/requests/timeline.ts

```diff
@@ -81,8 +124,10 @@ export class TimelineItem
   // eslint-disable-next-line class-methods-use-this
   toFirestore(modelObject: TimelineItem): ITimelineItem {
     return {
+      actorSnapshot: modelObject.actorSnapshot,
       offerSnapshot: modelObject.offerSnapshot,
       requestSnapshot: modelObject.requestSnapshot,
+      action: modelObject.action,
       createdAt: modelObject._createdAt,
     };
   }
```

Comment:
The file in the front end should follow the same design as the backend. We should be using `toObject`. The code split before these could be aligned. Until we can extract them, we should keep them aligned (using the `backend` one as the example).

Reply:
Done by 4788542